lib/process-data-availability.js

```diff
@@ -2,6 +2,9 @@ const { getLookupMap, getStandardMetadata } = require('./common');
 const lookupMap = getLookupMap();
 const standardMetadata = getStandardMetadata();
 
+// This is because a resource name and a lookup value can be the same. For eg both can be 'Property'
```

Comment:
I think this can be handled a different way. 

The cache can have expansions under the parent resource, similar to the rollups. 

If something has a parent resource, then the cache would be similar to this:

`cache[parentResourceName].expansions[resourceName][fieldName][lookupValue]`